CMakeLists.txt

```diff
@@ -53,6 +53,9 @@ add_env_executable(EXE flake8 REQUIRED)
 add_env_executable(EXE vvp REQUIRED)
 add_env_executable(EXE iverilog REQUIRED)
 add_env_executable(EXE xcfasm REQUIRED)
+add_env_executable(EXE xc7frames2bit REQUIRED)
+add_env_executable(EXE bitread REQUIRED)
+add_env_executable(EXE prjxray-config)
 
 get_target_property_required(PYTHON3 env PYTHON3)
```

Comment:
Should remove `third_party/prjxray` on line 65 below?

Reply:
prjxray is still needed as it contains all the various python scripts to access the db. The prjxray-tools conda-package instead contains only the bit tools

Reply:
I didn't say remove the submodule, I said from `CMakeLists.txt`.  Do you believe there are any remaining CMake targets used from `third_party/prjxray`?

Reply:
Aha, my bad, I misunderstood. Anyway no, looking at the CMake targets in prjxray, it seems that those are only needed to build the tools, therefore yes, that line can be removed.

Reply:
Fixed